docs/integration/resources.md

```diff
@@ -0,0 +1,240 @@
+DTS is part of an effort to harmonize how our community searches for, obtains,
+and cites scientific data, but the size and diversity of this community make it
+difficult to find an answer even for our purposes alone!
+
+Nevertheless, we press on. The metadata specification used by the DTS is likely
+to undergo changes of all sorts as we figure things out, but we'll give our best
+effort to updating documentation as these changes occur.
+
+### DTS Metadata Specification
+
+The DTS stores file metadata in the [Frictionless DataResource](https://specs.frictionlessdata.io/data-resource/)
+format. This format is simple and relatively unopinionated, and allows for
+additional fields as needed. The most important elements of the Frictionless
+DataResource type are:
+
+* `name`: the name of a resource (file)
+* `path`: the path of the resource relative to the root directory of the
+  [staging area](staging_area.md) in which you make the file available
+
+The following fields are used by the DTS in addition to the ones described in
+the link:
+
+* `id`: your organization's unique identifier for the resource
+* `credit`: credit metadata associated with the resource that conforms to the
+  [KBase credit metadata schema](https://github.com/kbase/credit_engine)
+* `metadata`: an optional unѕtructured field that you can use to stash
+  additional information about the resource if needed. Don't worry if you can't
+  think of a use for this one.
+
+_It might be good to show an example here._
+
+By the way, the DTS uses the [Frictionless DataPackage](https://specs.frictionlessdata.io/data-package/)
+format to store file manifests for bulk file transfers. A data package is just
+a collection of data resources with some additional metadata that applies to
+all of them. A file manifest is generated automatically by the DTS after each
+successful transfer.
+
+If you adopt the Frictionless DataResource format for your own file metadata,
+integration with the DTS will be very easy. If your organization already has its
+own metadata format, the DTS team can work with you to determine how it can be
+translated to the Frictionless format for use by the DTS.
+
+## Endpoint Recommendations
+
+Create a REST endpoint that accepts an HTTP `GET` request with a list of unique
+file identifiers specified (for example) by a comma-separated set of request
+parameters. This endpoint responds with a body containing a JSON list of
+objects representing [Frictionless DataResources](https://specs.frictionlessdata.io/data-resource/)
+describing the requested files in as much detail as is practical.
+
+Error codes should be used in accordance with HTTP conventions:
+
+* A successful query returns a `200 OK` status code
+* An improperly-formed request should result in a `400 Bad Request` status code
+* If one or more file IDs do not correspond to existing files in your
+  organization's database, their entries in the JSON list in the response should
+  be set to `null`.
```

Comment:
Sensible approach! I hate it when a batch endpoint returns an error if one or more inputs is invalid... and especially if it dies on the first error, rather than checking all inputs and returning a list of errors.